x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -109,6 +115,8 @@ export const createMetricThresholdExecutor = (libs: InfraBackendLibs) =>
     // no data results on groups that get removed
     const groups = [...new Set([...prevGroups, ...resultGroups])];
 
+    const isNotUngrouped = !isEqual(groups, [UNGROUPED_FACTORY_KEY]);
```

Comment:
This is already a lot easier for me to parse. Can we get rid of this double negative? Maybe just call this `hasGroups` or `groupsExist` ?